taipy/gui/gui.py

```diff
@@ -411,6 +419,32 @@ def __init__(
             for library in libraries:
                 Gui.add_library(library)
 
+    def __load_scripts(self, script_paths: t.Optional[t.Union[str, Path, t.List[t.Union[str, Path]]]]):
+        def load_script(path: t.Union[str, Path]):
```

Comment:
what's the point of inner defined function ?

Reply:
it was based on Fabien suggestion

Reply:
then ...

Reply:
Inner functions don't clobber the namespace. I like that, as long as these functions are not evaluated over and over... which is the case here.

Reply:
the code of the inner function is called only once ... what's the point ?

Reply:
Oh... that's different. 
A function called only once is kind of useless indeed.
Then a comment does the trick, expanding the function where it belongs.